experiment/simulation/js/main.js

```diff
@@ -1 +1,150 @@
-//Your JavaScript goes in here
+'use strict';
+
+document.addEventListener('DOMContentLoaded', function () {
+
+	const playButton = document.getElementById('play');
+	const pauseButton = document.getElementById('pause');
```

Comment:
Play and pause should be the same button as both of them will not be required at the same time